internal/config/default.yaml

```diff
@@ -5,5 +5,9 @@ MongoDB:
   host: localhost
   name: things_db
 
+Cloud:
+  host: localhost
+  port: 8182
```

Comment:
Is that the correct port for babeltower service? Also suggest to use the service hostname (I think it's `bt`), what do you think?

Reply:
Opsie, nice observation! Updated it :smile: 
Also, added `development` configurations for the cloud.

About changing `localhost` to `bt`, I'm not sure. For `bt` to work, the user would have to edit the DNS file so that `bt` points to `localhost`. When we deploy with `knot-cloud` and `docker swarm`, that's a different scenario, and we explicitly instruct the user to edit their `DNS` config file. 

What do you think @joaoaneto?

Reply:
I think `default` configuration should be neutral apart from deploying specificities. So, `localhost` and `80` fits well.

Reply:
Done :smile: